app/services/ccms/case_add_requestor.rb

```diff
@@ -458,9 +458,11 @@ def provider
       @provider ||= @legal_aid_application.provider
     end
 
+    # :nocov:
     def lead_proceeding
       @lead_proceeding ||= @legal_aid_application.lead_proceeding
     end
+    # :nocov:
```

Comment:
If you add a `legal_aid_application` getter method, the methods `applicant`, `provider`, `lead_proceeding`, `proceedings`, `vehicle`, and `wage_slips` can be delegated to `legal_aid_application` and some complication will be removed and you won't have to `nocov` this method.

```ruby
attr_reader :legal_aid_application
delegate :applicant, :provider, :lead_proceeding, :proceedings, :vehicle, :wage_slips, to :legal_aidd_application
```

Reply:
Actually, it's no longer needed.  It's always true for MVP, because we only ever have one proceeding per applciation.